### PR TITLE
add invaliditydate class for crl entry extensions

### PR DESCRIPTION
**Pull request by reaperhulk**
refs #2533

### CODE REVIEW

File: src/cryptography/x509/extensions.py [hunk 1]
```diff
@@ -1001,3 +1002,30 @@ def __ne__(self, other):
...
+
+    def __eq__(self, other):
+        if not isinstance(other, InvalidityDate):
+            return NotImplemented
+
+        return self.invalidity_date == other.invalidity_date
+
+    def __ne__(self, other):
+        return not self == other
+
```
Comment (alex) [inline src/cryptography/x509/extensions.py]: Should we just do `__hash__` while we're here?

**Comment by alex** [comment]
jenkins, retest this please

**Comment by alex** [comment]
jenkins, retest this please

**Comment by reaperhulk** [comment]
jenkins, retest this please (lousy network)

**Pull request merged**